clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/evaluation/EventAdapter.kt

```diff
@@ -81,11 +81,20 @@ class EventAdapter(
         return items
             .filterNotNull()
             .map { productMap: Map<String, Any> ->
-                val normalisedMap = productMap.map {
-                    Utils.getNormalizedName(it.key) to it.value
-                }.toMap()
 
-                TriggerValue(normalisedMap[Utils.getNormalizedName(propertyName)])
+                var op = productMap[propertyName]
```

Comment:
Can't we replace logic L85 to L95 with `evaluateActualPropertyValue()`. Seems duplicate code.

Reply:
this bit is for the items from charged event product, the method you have mentioned is for event parameters (event properties).

Reply:
<img width="1307" alt="Screenshot 2024-11-29 at 12 12 00 PM" src="https://github.com/user-attachments/assets/41ec9589-d208-4176-82ca-4afa027438ab">
As per screenshot we can see duplicate code can be replaced by single method. Specific Map can be passed to method.